packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -597,6 +601,111 @@ it('should not show sort icon if disabled in column level', () => {
   ).toBeFalsy();
 });
 
+it('should display correct sort icons for ascending first', async () => {
```

Comment:
Bad merge or what?

Reply:
I think this was added because this branch is updated with main changes (not latests though)